fractal_server/app/routes/api/v2/task_collection_custom.py

```diff
@@ -112,7 +112,7 @@ async def collect_task_custom(
         package_root = Path(task_collect.package_root)
 
     # Set task.owner attribute
-    owner = user.username or user.slurm_user
+    owner = user.username or user.settings.slurm_user
```

Comment:
make sure you include `verify_user_has_settings`, before calling `user.settings`